STACpopulator/input.py

```diff
@@ -23,18 +24,13 @@ def __init__(self) -> None:
         self.links = []
 
     @abstractmethod
-    def __iter__(self) -> Iterator:
+    def __iter__(self) -> Iterator[Tuple[str, str, MutableMapping[str, Any]]]:
         """
         Iterate over this loader.
 
-        Returns items from the input. Items may be anything depending on the specific concrete
```

Comment:
Items may not be anything since the populator ingest functions expect a tuple with 3 items.